src/references/events_hooks.md

```diff
@@ -0,0 +1,433 @@
+
+    ```lua linenums="1"
+    local function OnPlayerLeft(player)
+        print(player.name .. " has left the game")
+    end
+
+    Game.playerLeftEvent:Connect(OnPlayerLeft)
+    ```
+
+    Knowing when a player has left the game can be used in many ways. From updating the player's **Storage**, to cleaning up variables and tables that may hold references to the **Player** leaving the game.
+
+See [Game](../api/game.md#events) API for more information.
+
+### IK Anchor
+
+**IKAnchors** are objects that can be used to control player animations. They can be used to specify the position of a specific hand, foot, or the hips of a **Player**. **IKAnchors** have events that can be listened too to detect when an **IKAnchor** is activated on a player, and deactivated from the player.
+
+See [IKAnchors](../api/ikanchor.md#events) API for more information.
+
+### Player
+
+**Player** is an object representation of the state of a player connected to the game, as well as their avatar in the world. The **Player** object contains a lot of useful events that can be listened too. For example, a useful thing to know is what key binding the **Player** has pressed. Listening to the `bindingPressedEvent` when a **Player** presses a specific key binding is an easy way to do this.
+
+```lua linenums="1"
+local localPlayer = Game.GetLocalPlayer()
+
+localPlayer.bindingPressedEvent:Connect(function(player, binding)
+    print("The player pressed " .. binding)
+end)
+```
+
+In the code above, when a player presses a binding, it will print out to the **Event Log** what the binding was. This is useful if you need to perform a specific action based on what binding the **Player** has pressed.
+
+See [Player](../api/player.md#events) and [Bindings](../api/key_bindings.md) API for more information.
+
+### Trigger
+
+A **Trigger** is an invisible and non-colliding **CoreObject** which fires events when it interacts with another object (e.g. A **Player** walks into it). **Triggers** can be used in a wide range of cases, one of them being a way to detect when a **Player** has entered the **Trigger** to show an interaction label to open a door.
+
+```lua linenums="1"
+local TRIGGER = script:GetCustomProperty("pickupTrigger")
+
+TRIGGER.beginOverlapEvent:Connect(function(trigger, obj)
+    if(Object.IsValid(obj) and obj:IsA("Player")) then
+        print("Player entered the trigger").
+    end
+end)
+
+TRIGGER.endOverlapEvent:Connect(function(trigger, obj)
+    if(Object.IsValid(obj) and obj:IsA("Player")) then
+        print("Player left the trigger").
+    end
+end)
+```
+
+In the example code above, the listeners will display a message in the **Event Log** when the **Player** enters and exits the **Trigger**.
+
+!!! tip
+    A **Trigger** will also work in a **Client Context** by setting the **Collision** property to **Force On**.
+
+!!! warn
```

Comment:
They keyword is `warning`
```suggestion
!!! warning
```

In general, I'm not too much of a fan of having to admonition boxes right after each other, maybe we can have one above the code instead? 

Also, if you don't want to give admonitions a title, you can put the text in quotes and it will collapse for a more compact look like so:

`!!! warning "When a **Vehicle** enters or exits a **Trigger**, it will fire the listener twice."`

![image](https://user-images.githubusercontent.com/58569561/129583612-33ea76ae-1585-46bf-9c12-4e965b0be565.png)